README.md

```diff
@@ -46,6 +46,7 @@ Usage: cli [options] <asyncapi> <template>
 Options:
   -V, --version                  output the version number
   -d, --disable-hook <hookName>  disable a specific hook
+  --debug                        enable more specific errors about filters. Keep in mind that as a result errors about template are less descriptive
```

Comment:
I like that we have a `--debug` option but it shouldn't be **only** about filters. We can either remove the "specificity" on the description or call it `--debug-filters`. I'm more inclined to the first option and create an issue to follow up on improving debugging. Probably using https://www.npmjs.com/package/debug.

Reply:
How about `enable more specific errors in the console. At the moment it only shows specific errors about filters. Keep in mind that as a result errors about template are less descriptive`

I'm with option to leave `--debug` and then extend it with more use cases.

Reply:
Sounds good for now. What worries me is this "for now". We soon should be adding error handling to the generator because logging an error in the console is fine when you use it as a CLI but it's not great when you use it as a library. I'd just remove any mention to filters here and would probably put this information somewhere else but I'm also concerned that people will not see it if it's not here. It's just weird to me that a "debug" option makes explicit mention to one of the features.

Reply:
the description reflects reality, and I think we need it there so there are no other expectations. Once we add more debugging options we should simply adjust the description to make it more generic, right? It is strange that it is for filters only but this is because it is just a first iteration

Reply:
Ok. Go for it! 👍

Reply:
@fmvilas description updated